app/templates/components/orders/ticket-price.hbs

```diff
@@ -1,5 +1,11 @@
-{{#if (eq ticket-type 'free')}}
+{{#if (eq @ticket.type 'free')}}
     {{t "Free"}}
 {{else}}
-    {{display-symbol}} {{display-price}}
+    {{#if (eq @display 'price')}}
+        {{currency-symbol @eventCurrency}} {{format-money @ticket.price}}
+    {{else if (eq @display 'subTotal')}}
+        {{currency-symbol @eventCurrency}} {{format-money @ticket.subTotal}}
+    {{else if (eq @display 'order-subTotal')}}
+        {{currency-symbol @eventCurrency}} {{format-money (mult (sub @ticket.price @ticket.discount) (ticket-attendees @attendees @ticket.attendees))}}
+    {{/if}}
```

Comment:
Why are you making it so so difficult? You just have to check the type and show either free or passed value. Pass the amount as an argument and don't do these complex calculations here. I don't know why eceryone likes so much complicated solutions. Passing attendees and multiplying and switch casing. I don't know what to say except changing it myself will take much much less time than reviewing this

Reply:
@iamareebjamal sorry sir.